modules/system/applications.nix

```diff
@@ -1,18 +1,60 @@
-{ config, lib, pkgs, ... }:
-
-with lib;
-
-let
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
 
-  cfg = config.system;
+{
+  config = {
+    system.checks.text = lib.mkAfter ''
+      ensureAppManagement() {
+        for appBundle in /Applications/Nix\ Apps/*.app; do
+          if [[ -d "$appBundle" ]]; then
+            if ! touch "$appBundle/.DS_Store" &> /dev/null; then
```

Comment:
does this break code signatures?

Reply:
Just tested this with Apparency, code signatures and notarization are still fine after touching `.DS_Store`